components/teams-service/cmd/teams-service/main.go

```diff
@@ -142,6 +162,16 @@ Please pass a config file as the only argument to this command.`))
 	if err != nil {
 		fail(errors.Wrap(err, "could not initialize storage"))
 	}
+	err = project_purge.RegisterTaskExecutors(manager, "teams", service.Storage)
+	if err != nil {
+		fail(errors.Wrap(err, "could not register project purge client"))
+	}
+
+	err = manager.Start(context.Background())
```

Comment:
I suppose this is where the teams-service starts its project purge, but I don't see any new code in teams-service to actually do the project purge. Can you explain what is happening?
(And similar question for the domain services--I don't see any files in the PR to instrument them at all.)

Reply:
There is a generic TaskExecutor implemented in the project_purge package. We pass it an interface that actually knows how to do the purging. See line 165 of this file.